docs/gameobject_questitem.md

```diff
@@ -2,6 +2,6 @@
 
 Column | Type | Description
 --- | --- | ---
-GameObjectEntry | int(10) unsigned | Entry ID of a gameobject that can be looted only while the related quest is active.
+GameObjectEntry | int(10) unsigned | Entry ID of a gameobject that contains quest items only while the related quest is active.
```

Comment:
```suggestion
GameObjectEntry | int(10) unsigned | Entry ID of a gameobject that contains quest items when the related quest is active.
```

Reply:
better, although the rest of them say "only while"

Reply:
well you can edit those too :D but thank you for PRs!!

to edit directly in github you can go here https://github.com/azerothcore/wiki/pull/110/commits then click the "..." and "edit file" and you " Commit directly to the patch-3 branch. "

Reply:
nice, thanks for the tip